HoneyBadgers.ConsoleApp/Repositories/MovieRepository.cs

```diff
@@ -84,5 +85,121 @@ private string StringValidation(int minLength, int maxLength)
+            }
+            Console.WriteLine($"Current writer is: {selectedMovie.Writer}. Type writer or press enter to continue without changes");
+            var writer = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(writer.Trim()))
+            {
+                selectedMovie.Writer = writer;
+            }
+            Console.WriteLine($"Current actors are: {selectedMovie.Actors}. Type new actors or press enter to continue without changes");
+            var actors = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(actors.Trim()))
+            {
+                selectedMovie.Actors = actors;
+            }
+            Console.WriteLine($"Current plot is: {selectedMovie.Plot}. Type new plot or press enter to continue without changes");
+            var plot = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(plot.Trim()))
+            {
+                selectedMovie.Plot = plot;
+            }
+            Console.WriteLine($"Current genre is: {selectedMovie.Genre}. Type new genre or press enter to continue without changes");
+            var genre = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(genre.Trim()))
+            {
+                selectedMovie.Genre = genre;
+            }
+            Console.WriteLine($"Current country is: {selectedMovie.Country}. Type new country or press enter to continue without changes");
+            var country = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(country.Trim()))
+            {
+                selectedMovie.Country = country;
+            }
+            Console.WriteLine($"Current rating is: {selectedMovie.ImdbRating}. Type new rating or press enter to continue without changes");
+            do
+            {
+                var ratingInput = Console.ReadLine();
+                double rating;
+                bool wasparsedSuccessfully = double.TryParse(ratingInput, out rating);
+                if (wasparsedSuccessfully == false)
+                {
+                    Console.WriteLine("Something went wrong! \nYou have not entered a numeric value.");
+                    break;
+                }
+                if (rating > 10)
+                {
+                    Console.WriteLine("Rating value can't be higher than 10.");
+                }
+                if (rating < 0)
+                {
+                    Console.WriteLine("Rating value can't be a negative value.");
+                }
+                else
+                {
+                    selectedMovie.ImdbRating = rating;
+                }
+            } while (true);
+        }
+        public void MovieDataEdition()
+        {
+            Console.WriteLine("Would you like to edit any user data?");
+            Console.WriteLine("Press ENTER to print the list of users or any key to continue without...");
+
+            ConsoleKey choice = Console.ReadKey(true).Key;
+            if (choice == ConsoleKey.Enter)
```

Comment:
myślę że to powinno zadziałać: if (choice == ConsoleKey.Enter) { PrintMovies(); } i poza if czy else'm dać InputChangesInMovieData

Reply:
done